src/web/mjs/connectors/TopToon.mjs

```diff
@@ -34,11 +34,11 @@ export default class TopToon extends Connector {
 
     async _getChapters(manga) {
         const request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        const data = await this.fetchDOM(request, 'div.episode_list ul a.episode-items');
+        const data = await this.fetchDOM(request, 'div.eplist ul a.episode-items');
         return data.map(element => {
-            let title = element.querySelector('p.episode_title').textContent.trim();
-            const subtitle = element.querySelector('p.episode_stitle');
-            title += subtitle ? ' - ' + subtitle.textContent.trim() : '';
+            let title = element.querySelector('p.ep_title').textContent.trim();
+            const subtitle = element.querySelector('p.ep_stitle');
```

Comment:
Consider `element.querySelector('p.ep_stitle')?.textContent.trim()` to simplify the the following condition for ternary operator.

Reply:
I am dumb, i cant get past the lint error

Reply:
I wont do that, because while nightly electron version support that syntax `element.querySelector('p.ep_stitle')?`, master branch doesnt.